data-raw/generate-comp-data.R

```diff
@@ -0,0 +1,48 @@
+## This script generates the comparison data that is used in the old new tests (a bit by bit)
+## comparison. Whenever there is devlopment that should change Hector's output this script
+## should to generate new comparison data.
+## Since this script will only while developing the package devtools::load_all() and
+## file paths that are relative to the hector project instead of built package.
+
+# Load the Hector package
+devtools::load_all()
+
+# Create the output directory.
+out_dir <- here::here('tests', 'testthat', 'compdata')
+dir.create(out_dir, showWarnings = FALSE)
+
+# Determine the version of Hector that will generate the comparison data.
+# This will be saved to help keep track of the last time the comparison data
+# was actually updated.
+session_info   <- sessionInfo(package = 'hector')
+hector_version <- session_info$otherPkgs$hector$Version
```

Comment:
`packageVersion('hector')` ?

Reply:
Would we want to track git git commit as well?

Reply:
Oh yes, that is a great idea!